example/eblob_backend.c

```diff
@@ -1140,7 +1151,10 @@ static void dnet_blob_config_cleanup(struct dnet_config_backend *b)
 {
 	struct eblob_backend_config *c = b->data;
 
-	eblob_backend_cleanup(c);
+	if (c->eblob)
+		eblob_backend_cleanup(c);
+
+	free(c->data.file);
 }
```

Comment:
This requires a long description of how is it possible, that we have half-filled control structure, but did not initialize eblob. Including two-steps init process (parse config, then call ->init()), and that parsing disabled backend configs will not have second stage, but we will have to cleanup resources allocated during config parsing.